crates/unftp-auth-rest/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: help
+help: # Shows available `make` commands
+	@echo 'Available `make` commands:' >/dev/stderr
+	@echo >/dev/stderr
+	@awk -F'#' '/^[a-z][A-Za-z0-9]+/ {if (NF > 1) { sub(/:[^#]*/, ""); print $$1 "\t\t" $$2}}' Makefile
+
+.PHONY: docs
+docs: # Creates the API docs and opens it in the browser
+	cargo doc --all-features --no-deps --open
+
+.PHONY: pr-prep
+pr-prep: # Runs checks to ensure you're ready for a pull request
+	cargo fmt --all -- --check
+	cargo clippy --all-features --all-targets
+	cargo test --all-features --all-targets
+	cargo test --doc
+	cargo build --all-features --all-targets
+	cargo doc --all-features --no-deps
+
+.PHONY: publish
+publish: # Publishes the lib to crates.io
+	cargo publish --all-features --verbose
```

Comment:
question(non-blocking): what about removing the `--all-features` flag? it looks to me that this cargo file doesn't have any features, so this is unnecessary.

Reply:
yup, will remove

Reply:
Fixed